examples/layers/JSONLayers/WORLD_DTM.json

```diff
@@ -0,0 +1,69 @@
+
+{
+    "type":       "elevation",
+    "protocol":   "wmts",
+    "id":         "MNT_WORLD_SRTM3",
+    "url":        "http://wxs.ign.fr/va5orxd0pgzvq3jxutqfuy0b/geoportail/wmts",
+    "noDataValue": -99999,
+    "updateStrategy": {
+        "type": 1,
+        "options": {
+            "groups": [3, 7, 10]
```

Comment:
may be use 
for WORLD
`"groups": [3, 7]`
and for HIGH RES
`"groups": [10, 14]`